FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/FreeRTOS_Sockets.c

```diff
@@ -1397,6 +1397,12 @@ FreeRTOS_Socket_t *pxSocket;
 	( void ) uxOptionLength;
 
 	configASSERT( xSocket != NULL );
+	
+	if (pxSocket == FREERTOS_INVALID_SOCKET)
+	{
+		xReturn = FREERTOS_EINVAL;
+		return xReturn;
+	}
```

Comment:
Firstly, mind the white space:
~~~c
- if (pxSocket == FREERTOS_INVALID_SOCKET)
+ if( pxSocket == FREERTOS_INVALID_SOCKET )
~~~
I think it is good to treat the a NULL pointer the same, returning a `-EINVAL`:
~~~c
if( ( pxSocket == NULL ) || ( pxSocket == FREERTOS_INVALID_SOCKET ) )
{
    xReturn = FREERTOS_EINVAL;
    return xReturn;
}
~~~

Reply:
Oops about the white space. IDE adds them sometimes!
Also, thanks for the pointer about pxSocket being NULL. I made a commit to correct that.